src/js/popup.js

```diff
@@ -507,7 +507,9 @@ Logic.registerPanel(P_CONTAINERS_LIST, {
     });
 
     Logic.addEnterHandler(document.querySelector("#edit-containers-link"), () => {
-      Logic.showPanel(P_CONTAINERS_EDIT);
+      if (!document.querySelector("#edit-containers").classList.contains("disable-edit-containers")){
```

Comment:
```suggestion
      if (!e.target.classList.contains("disable-edit-containers")){
```

If you add the class to the link instead I think we simplify this greatly.